### PR TITLE
fix(login): added login subtitle

### LINKED ISSUE #1598: Add header description to login component
Core PR - https://github.com/patternfly/patternfly-next/pull/1529

adds optional `.pf-c-login__main-header-desc`

### PR DESCRIPTION
**Pull request by dtaylor113**
Fixes #1598 

image

### CODE REVIEW

**Comment by dtaylor113** [comment]
Hi @redallen,
I was able to add the 'popup example' link to the Login example.  It now appears under 'Demos':
image
Some Questions:
1. I still do not know what is causing the `SyntaxError`, it's something in the `render()` function.
2. Is there a way to have the Login appear under components with the props, and just have the Login 'popup example' appear under Demos?  
- Thanks

**Comment by redallen** [comment]
1. [Buble](https://github.com/bublejs/buble) is what is complaining about something in your render() function. Your code gets some regexs run on it before being handed to Buble in [`liveEdit.js`](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-docs/src/components/componentDocs/liveEdit.js). Print `toParse` and then pass it to Buble via `require( 'buble' ).transform(toParse)` to debug.
2. Not quite sure if you want it in both places, only one place, or to have a "popout demo" while remaining under components. All are possible.

**Comment by dtaylor113** [comment]
> 2\. Not quite sure if you want it in both places, only one place, or to have a "popout demo" while remaining under components. All are possible.

 I'd like to have a "popout demo" while remaining under Components  -ie. not use Demo section at all

**Comment by redallen** [comment]
Try removing the `section: "demos"` and let me know if the popout doesn't work!

**Comment by dtaylor113** [comment]
> Try removing the `section: "demos"` and let me know if the popout doesn't work!

That puts 'Login Page' at top level, under 'Demos' (not in Demos or Components).

**Comment by dtaylor113** [comment]
> 2\. Not quite sure if you want it in both places, only one place, or to have a "popout demo" while remaining under components. All are possible.

Changing it to `section: "components"` put it under the Components section -thanks

**Comment by amarie401** [comment]
Hi @dtaylor113 

I will let @tlabaj & @jschuler take this review as I haven't touched the login page in a long time, looks good to me otherwise though  :)

**Comment by dtaylor113** [comment]
Hi @jschuler, @tlabaj, any progress on this review?  -thanks
Pls let me know if I identified the correct reviewers, I'm not sure who should review what.
Thanks,
- Dave

**Comment by dtaylor113** [comment]
Hi @jschuler, @tlabaj, pls let me know if there is another dev I should add to the Reviewers list?
Thanks :-)

File: packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ---
 title: "Login Page"
 cssPrefix: "pf-c-login"
+section: "components"
+seperatePages: true
 ---
 ## Simple Login Page
+<a href="SimpleLoginPage" target="_blank">Popout example</a>
+
 ```js
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md]: Change to ```nolive to not render the broken preview.

Comment (dtaylor113) [reply packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md]: Hi @redallen, when I do that I get:
image

Comment (redallen) [reply packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md]: Looks good! No more confusing errors trying to render a page there that should really be fullscreen.

**Comment by dtaylor113** [comment]
Thanks @redallen, @tlabaj or @jschuler can you please review?  -thanks

**Comment by mcarrano** [comment]
LGTM! @mceledonia can you verify? https://1696-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/loginpage/simpleloginpage/

**Comment by mceledonia** [comment]
LGTM as well!

File: packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.d.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface LoginPageProps extends HTMLProps<HTMLElement> {
   footerListItems?: ReactNode;
   footerListVariants?: OneOf<typeof LoginListVariant, keyof typeof LoginListVariant>;
   loginTitle: string;
+  loginSubtitle: string;
```
Comment (dgutride) [inline packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.d.ts]: Please make this property optional

Comment (dtaylor113) [reply packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.d.ts]: Ok, updated, thanks

**Pull request merged**